store/models.py

```diff
@@ -2,12 +2,16 @@
 from django.contrib.auth.models import User
 from django.core.validators import MaxValueValidator, MinValueValidator
 
+from store.store_utils import StoreUtils
+
 
 class Store(models.Model):
     name = models.CharField(max_length=100)
     owner = models.ForeignKey(User, on_delete=models.CASCADE)
+    category = models.CharField(max_length=30)
     city = models.CharField(max_length=25)
     state = models.CharField(max_length=25)
+    image = models.ImageField(upload_to=StoreUtils.generate_store_image_path)
```

Comment:
consider if this feild can be nullable or blank. a store might add an image later.